wlms/client.go

```diff
@@ -41,6 +41,13 @@ const (
 	RECENTLY_DISCONNECTED
 )
 
+// The protocol versions supported by the metaserver
+const (
+	BUILD19 int = 0
+	BUILD20     = 5
```

Comment:
No "int"?

Reply:
Good catch. I thought I don't need one there based on what similar structures are doing. But reading about it it seems that I should have the int after all.